app/views/reports/index.html.haml

```diff
@@ -18,21 +18,21 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#reporting-container.col-lg-10.col-lg-offset-1
-  - @counter = 0
-  #report-selection.text-center
-    .page-header
-      %h1
-        = t(:reporting)[:index][:intro]
-        %br
-        %small
-          = t(:reporting)[:index][:subtext]
-    - if !Setting.get_value('use_short_interaction')
-      - $canned_reports.delete("ShortInteractionsReport")
-    - $canned_reports.sort.each do |report|
-      = link_to report.constantize.title, setup_reports_path(report: report), method: :get, remote: true, class: "btn btn-primary btn-lg"
-      - @counter += 1
-      - if @counter.modulo(4) == 0
-        %br
-        %br
-  #report-container
+#reporting-container.container
```

Comment:
Can you check on the usage of the `container` class here? We're not using `container`s anywhere else